pom.xml

```diff
@@ -81,6 +91,33 @@
 
     <build>
         <plugins>
+            <plugin>
+                <groupId>org.teavm</groupId>
+                <artifactId>teavm-maven-plugin</artifactId>
+                <version>0.9.2</version>
+                <dependencies>
+                    <dependency>
+                        <groupId>org.teavm</groupId>
+                        <artifactId>teavm-classlib</artifactId>
+                        <version>0.9.2</version>
+                    </dependency>
+                </dependencies>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>compile</goal>
+                        </goals>
+                        <phase>process-classes</phase>
+                        <configuration>
+                            <mainClass>com.shapesecurity.salvation2.Salvation2</mainClass>
+                            <minifying>true</minifying>
+                            <sourceMapsGenerated>false</sourceMapsGenerated>
+                            <sourceFilesCopied>false</sourceFilesCopied>
+                            <targetFileName>salvation-v${project.version}.min.js</targetFileName>
```

Comment:
I would not include the project version in the file name; that makes it harder to consume, and I don't think there's much advantage to it.

Reply:
I was thinking about it from a consumer's standpoint. Generally the files are versioned, of course, but I don't have a good argument for versioning in this case 😆